css/style.css

```diff
@@ -229,27 +236,35 @@ ul {
 .table__forma {
     display: flex;
     padding-bottom: 40px;
-    margin-left: 180px;
+    margin-left: 200px;
 }
 
 .forma+.forma {
-    margin-left: 20px;
+    margin-left: 40px;
 }
 
 .forma {
     width: 140px;
-    height: 120px;
     margin-left: -20px;
 }
 
 .forma__title {
-    margin: 0;    
+    margin: 0;
+    width: 140px;
     font-size: 16px;
     font-weight: 600;
     color: rgb(0, 0, 0);
     line-height: 1.125;
 }
 
+.forma__title-cell {
+    margin: 0;
+}
+
+.forma__title-cell3{
+    margin-top: 16px;
+}
+
```

Comment:
What are non-header related changes doing here? Incorrect class naming .forma__title-cell3

Reply:
Changes back to originals